ros2cli/ros2cli/cli.py

```diff
@@ -53,6 +63,18 @@ def main(*, script_name='ros2', argv=None, description=None, extension=None):
     # parse the command line arguments
     args = parser.parse_args(args=argv)
 
+    if not args.use_python_default_buffering:
+        # Make the output always line buffered.
+        # TextIoWrapper has a reconfigure() method, call that if available.
+        # https://docs.python.org/3/library/io.html#io.TextIOWrapper.reconfigure
+        try:
+            sys.stdout.reconfigure(line_buffering=True)
+        except AttributeError:
+            # if stdout is not a TextIoWrapper instance, or we're using python older than 3.7,
+            # force line buffering by patching print
+            builtins.print = functools.partial(print, flush=True)
+            pass
```

Comment:
Nit: This `pass` is unnecessary.

Reply:
Nice catch, fixed in https://github.com/ros2/ros2cli/pull/659/commits/6c5a30e0ecc9ba90bfe27e1b924a236683318d8d.